package.json

```diff
@@ -7,6 +7,7 @@
     "build": "next build",
     "start": "next start -H 0.0.0.0 -p ${PORT:-8080}",
     "db": "prisma studio",
+    "db:up": "docker run --name resilience-web -e POSTGRES_PASSWORD=password -p 5432:5432 -d postgres",
```

Comment:
Nice, this is really useful!

Reply:
Updated to use a more configurable docker-compose file